scripts/zones/Southern_San_dOria/npcs/Phillone.lua

```diff
@@ -31,7 +32,32 @@ end;
 -----------------------------------
  
 function onTrigger(player,npc) 
-    player:startEvent(0x01D);
+
+    local ATimelyVisit = player:getQuestStatus(SANDORIA, A_TIMELY_VISIT);
+    local ATimelyVisitProgress = player:getVar("ATimelyVisitProgress");
+
+    -- printf("ATV %d, ATVP %d\n", player:getQuestStatus(SANDORIA, A_TIMELY_VISIT), player:getVar("ATimelyVisitProgress"));
+
+    if (ATimelyVisit == 1 and ATimelyVisitProgress == 4) then
+        player:startEvent(0x001a);  -- sorry I was crazy
+    elseif (ATimelyVisit == 1 and ATimelyVisitProgress == 7) then
+        player:startEvent(0x004e);  -- my house!
+    elseif (ATimelyVisit == 1 and ATimelyVisitProgress == 8) then
+        player:startEvent(0x005b);  -- find knight to save my family
+    elseif (ATimelyVisit == 1 and ATimelyVisitProgress == 9) then
+        player:startEvent(0x0030);  -- go get scepter in forest
+    elseif (ATimelyVisit == 1 and ATimelyVisitProgress == 11) then
+        player:startEvent(0x0015);  -- figuring everything out
+    elseif (ATimelyVisit == 1 and ATimelyVisitProgress == 12) then
+        player:startEvent(0x0031);  -- save my family at ordelles
+    elseif (ATimelyVisit == 1 and ATimelyVisitProgress == 13) then
+        player:startEvent(0x000e);  -- quest completion
+    elseif (ATimelyVisit == 2) then
```

Comment:
QUEST_AVAILABLE = 0;
QUEST_ACCEPTED  = 1;
QUEST_COMPLETED = 2;

Reply:
use the quest status variable in other locations where you have them as  raw numbers as well